lib/results-list.js

```diff
@@ -115,14 +115,34 @@ export const ResultItem = styled.span.attrs(() => ({ 'data-module': 'ResultItem'
   position: relative;
   padding: var(--space-1);
   text-decoration: none;
-  &:focus {
+  margin: var(--space-1) 0;
+
+  &:focus,
+  button:focus &,
+  a:focus & {
+    box-shadow: 0 0 0 1px white, 0 0 0 3px var(--colors-focused);
+    outline: none;
```

Comment:
In high contrast mode on Windows, keyboard users won't know what item is in focus because of the way the properties get overriden. [See this SO answer for details](https://stackoverflow.com/a/52616313/1592716). Can we swap out `outline: none` here for `outline-colour: transparent`?